src/literals.py

```diff
@@ -51,6 +51,7 @@
 MSG_WAITING_FOR_PEER = "waiting for peer relation"
 MSG_STATUS_DB_MISSING = "Opensearch connection is missing"
 MSG_STATUS_DB_DOWN = "Opensearch service is (partially or fully) down"
+MSG_STATUS_DB_UNHEALTHY = "Opensearch service is not in a green health state"
```

Comment:
I would keep this one more generic, perhaps along the lines of:
```suggestion
MSG_STATUS_DB_UNHEALTHY = "The OpenSearch service health is red."
```